tests/test_in_toto_run.py

```diff
@@ -38,8 +37,7 @@
 
 import in_toto.gpg.util
 
-# Suppress all the user feedback that we print using a base logger
-logging.getLogger().setLevel(logging.CRITICAL)
+
```

Comment:
I think this whitespace is one too many.

Reply:
This is where our code style guideline actually suggests [5 blank lines](https://github.com/secure-systems-lab/code-style-guidelines#blank-lines), which I find too much. I think 3 are a fair compromise...

Reply:
wow, TIL. Ok I guess...